metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/pipeline/loader/extractor/FileExtractorFromAnalysis.java

```diff
@@ -43,25 +43,22 @@
     private SraXmlParser<AnalysisType> sraXmlParser;
 
     public FileExtractorFromAnalysis(SraRetrieverByAccession sraRetrieverByAccession, SraXmlParser<AnalysisType>
-            sraXmlParser, Converter<AnalysisFileType,
-            File> fileConverter) {
+            sraXmlParser, Converter<AnalysisFileType, File> fileConverter) {
         this.sraRetrieverByAccession = sraRetrieverByAccession;
         this.sraXmlParser = sraXmlParser;
         this.fileConverter = fileConverter;
     }
 
-    public List<File> extractFilesFromAnalysis(List<String> analysisAccessions) {
+    public List<File> extractFilesFromAnalysis(String analysisAccession) {
         List<File> files = new ArrayList<>();
-        for (String analysisAccession : analysisAccessions) {
-            try {
-                String analysisXml = sraRetrieverByAccession.getXml(analysisAccession);
-                AnalysisType analysisType = sraXmlParser.parseXml(analysisXml, analysisAccession);
-                files.addAll(getFilesOfAnalysis(analysisType));
-            } catch (Exception exception) {
-                FILE_EXTRACT_SERVICE_LOGGER.log(Level.INFO, "Encountered Exception for analysis "
-                        + analysisAccession);
-                FILE_EXTRACT_SERVICE_LOGGER.log(Level.INFO, exception.getMessage());
-            }
+        try {
+            String analysisXml = sraRetrieverByAccession.getXml(analysisAccession);
+            AnalysisType analysisType = sraXmlParser.parseXml(analysisXml, analysisAccession);
+            files.addAll(getFilesOfAnalysis(analysisType));
+        } catch (Exception exception) {
+            FILE_EXTRACT_SERVICE_LOGGER.log(Level.SEVERE, "Encountered Exception for analysis "
```

Comment:
I thought if we load n number of analysis, just logging the exceptional analysis and later on inspecting them would be better, instead of stopping the process for each exceptional analysis.